### PR TITLE
radare2 0.10.2

### PR DESCRIPTION
**Pull request by nl5887**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Does your submission pass `brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Have you built your formula locally prior to submission with `brew install <formula>`?

### CODE REVIEW

**Comment by nl5887** [comment]
* bump to 0.10.2
* fixed style errors (Prefer to_s over string interpolation and Avoid comma)
* added test block

**Comment by ilovezfs** [comment]
I cannot seem to reproduce the build failure locally.

**Comment by tdsmith** [comment]
looks like it will be fixed by https://github.com/Homebrew/brew/pull/63

**Comment by ilovezfs** [comment]
Yup that fixed it.
```
==> brew --env
==> brew config
==> brew tests
==> brew tests --no-compat
==> brew readall --aliases --syntax
==> brew uses radare2
==> brew fetch --retry autoconf automake cairo fontconfig freetype gettext git glib gmp gobject-introspection libewf libffi libmagic libpng libtool lua51 makedepend openssl pcre pixman pkg-config swig vala valabind xz yara
==> brew fetch --retry radare2 --build-bottle
==> brew install --only-dependencies --verbose --build-bottle radare2
==> brew install --verbose --build-bottle radare2
==> brew audit radare2
==> brew bottle --verbose --rb radare2
==> brew bottle --merge --write --no-commit ./radare2-0.10.2.el_capitan.bottle.rb
==> brew uninstall --force radare2
==> brew uninstall --force autoconf automake cairo fontconfig freetype gettext git glib gobject-introspection libffi libpng libtool makedepend pixman pkg-config swig vala valabind xz
==> brew install ./radare2-0.10.2.el_capitan.bottle.tar.gz
==> brew test --verbose radare2
==> brew uninstall --force radare2
==> brew uninstall --force gmp libewf libmagic lua51 openssl pcre yara
iMac-TMP:homebrew-core joe$ 

```

**Comment by apjanke** [comment]
Homebrew/brew#63 is merged and the subsequent test-bot rerun on this worked. Think you can merge this now.

File: Formula/radare2.rb [hunk 1]
```diff
@@ -92,4 +93,8 @@ def install
       system "make", "install", *make_install_args
     end
   end
+
+  test do
+    system "#{bin}/r2", "--version"
```
Comment (ilovezfs) [inline Formula/radare2.rb]: Please have this assert_match against the version.

File: Formula/radare2.rb [hunk 2]
```diff
@@ -53,7 +53,7 @@ def install
     system "make", "install"
 
     resource("extras").stage do
-      ENV.append_path "PATH", "#{bin}"
+      ENV.append_path "PATH", bin.to_s
```
Comment (ilovezfs) [inline Formula/radare2.rb]: I don't think we need the `.to_s` at all.

File: Formula/radare2.rb [hunk 3]
```diff
@@ -62,7 +62,7 @@ def install
     end
 
     resource("bindings").stage do
-      ENV.append_path "PATH", "#{bin}"
+      ENV.append_path "PATH", bin.to_s
```
Comment (ilovezfs) [inline Formula/radare2.rb]: I don't think we need the `.to_s` at all.

File: Formula/radare2.rb [hunk 4]
```diff
@@ -3,17 +3,17 @@ class Radare2 < Formula
   homepage "http://radare.org"
 
   stable do
-    url "http://radare.org/get/radare2-0.10.1.tar.xz"
-    sha256 "5ac02717786f2ff3b5326927351d5ca38464da89675c8edfb4ded43addb22987"
+    url "http://rada.re/get/radare2-0.10.2.tar.xz"
```
Comment (ilovezfs) [inline Formula/radare2.rb]: The canonical links still appears to be
http://www.radare.org/get/radare2-0.10.2.tar.xz

File: Formula/radare2.rb [hunk 5]
```diff
@@ -3,17 +3,17 @@ class Radare2 < Formula
...
+    sha256 "1bc9ce6f5d6bec366324bc542653bba5c1b89a6980c17253ec3a1f75264beb3b"
 
     resource "bindings" do
-      url "http://radare.org/get/radare2-bindings-0.10.1.tar.xz"
-      sha256 "6cb91f4135b5e490185e25629850cb48c08c06882e2c870fa5ab1425cc84106f"
+      url "http://rada.re/get/radare2-bindings-0.10.2.tar.xz"
```
Comment (ilovezfs) [inline Formula/radare2.rb]: http://www.radare.org/get/radare2-bindings-0.10.2.tar.xz

File: Formula/radare2.rb [hunk 6]
```diff
@@ -3,17 +3,17 @@ class Radare2 < Formula
...
+      sha256 "7b47919c7f3d3a4eb432df8605160c72257a324c5f9e59c890799cf527228631"
     end
 
     resource "extras" do
-      url "http://radare.org/get/radare2-extras-0.10.1.tar.xz"
-      sha256 "c330210c8e6ce5fa8113c455e98c994fb5ecbb5d2f1c15c40c0d1bcbc24d5092"
+      url "http://rada.re/get/radare2-extras-0.10.2.tar.xz"
```
Comment (ilovezfs) [inline Formula/radare2.rb]: http://www.radare.org/get/radare2-extras-0.10.2.tar.xz

File: Formula/radare2.rb (hunk 2 above)
Comment (DomT4) [inline Formula/radare2.rb]: This will be Rubocop's fault _I suspect_. It nags to change it.

**Comment by nl5887** [comment]
ok, I've changed the urls to the canonical ones. The reason I used the short urls was because Radare is using them in all of there communication. 

Added assert_match on version, and removed the to_s.

File: Formula/radare2.rb [hunk 7]
```diff
@@ -92,4 +93,8 @@ def install
       system "make", "install", *make_install_args
     end
   end
+
+  test do
+    assert_match /radare2 0\.10\.2/, shell_output("#{bin}/r2 -version")
```
Comment (ilovezfs) [inline Formula/radare2.rb]: you can make it `version.to_s` so it doesn't need updating each time.

**Comment by ilovezfs** [comment]
other than the `version.to_s` change, :+1:

**Comment by apjanke** [comment]
:+1:

**Comment by tdsmith** [comment]
Thanks for your pull request! :tada: Homebrew thrives because of contributions like this one.

(Oops, forgot to squash those commits!)

**Pull request closed without merging**